dvc/output/base.py

```diff
@@ -220,3 +220,6 @@ def move(self, out):
 
         if self.scheme == "local" and self.use_cache and self.is_local:
             self.repo.scm.ignore(self.path)
+
+    def get_files_number(self):
+        return 0
```

Comment:
Note that this code is going to be used in non-local outputs too, so this is not right, it should be something like:
```
if self.use_cache and self.checksum:
    return 0

return 0
```